content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Pods And Endpoints Termination Flow
+content_type: tutorial
+weight: 60
+---
+
+
+<!-- overview -->
+
+## The Kubernetes model for connecting containers
+
+Now that you have a continuously running, replicated application, that is
```

Comment:
> Now that you have ...

Nothing has been discussed before this sentence or this paragraph, because this is the very first sentence in the page. The expression "Now that you have ..." reads pretty confusing to me.

Reply:
This wording comes from https://kubernetes.io/docs/tutorials/services/connect-applications-service/ (and needs fixing there too).